app/grandchallenge/participants/models.py

```diff
@@ -123,3 +125,81 @@ class RegistrationQuestionGroupObjectPermission(GroupObjectPermissionBase):
     content_object = models.ForeignKey(
         RegistrationQuestion, on_delete=models.CASCADE
     )
+
+
+class RegistrationQuestionAnswer(models.Model):
+    registration_request = models.ForeignKey(
+        RegistrationRequest,
+        on_delete=models.CASCADE,
+        related_name="registration_question_answers",
+    )
+    question = models.ForeignKey(
+        RegistrationQuestion,
+        on_delete=models.CASCADE,
+        related_name="answers",
+    )
+    answer = models.JSONField(default=str, blank=True)
+
+    class Meta:
+        unique_together = (("registration_request", "question"),)
+
+    @property
+    def empty_answer(self):
+        return self.answer == self._meta.get_field("answer").get_default()
+
+    def clean(self):
+        super().clean()
+
+        if self.empty_answer and self.question.required:
+            raise ValidationError(
+                f"The question {self.question.question_text!r} requires an answer"
+            )
+
+        if not self.empty_answer and self.question.schema:
+            try:
+                JSONValidator(schema=self.question.schema)(value=self.answer)
+            except ValidationError as e:
+                raise ValidationError(
+                    {"answer": f"Incorrect format: {e.__cause__}"}
+                )
+
+    def validate_constraints(self, exclude=None):
```

Comment:
May be better to use `*args, **kwargs` incase the function signature changes upstream.

Reply:
Since this isn't quite as core as the `save()` I am not sure: should a `return result` also be added? I've added it: better be save (hehe).